staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -78,6 +78,12 @@ type Options struct {
 	// See: https://openid.net/specs/openid-connect-core-1_0.html#IDToken
 	ClientID string
 
+	// APIAudiences are the implicit audiences that the API server identitifes
```

Comment:
Why "implicit"?

Reply:
Dropped. We've used implicit to refer to the server audience identifiers when to target audience is provided as part of the authentication request. It is unnecessary and confusing to include that here.